src/containers/SearchContainer/index.js

```diff
@@ -0,0 +1,213 @@
+/* eslint-disable no-underscore-dangle */
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { navigate } from '@reach/router';
+import { Box, Heading, Tabs, Tab, Text, TextInput } from 'grommet';
+import { Search as SearchIcon } from 'grommet-icons';
+
+import { Link, HighlightedText } from '../../components';
+import { useParams } from '../../hooks/use-params';
+
+const capitalize = text =>
+  text.substring(0, 1).toUpperCase() + text.substring(1);
+
+const categoryLabel = category =>
+  category === 'homepanels' ? 'Home' : capitalize(category);

```

Comment:
Might be a better practice to use:
```
const homepanels = 'homepanels';
const home = 'Home';
```

Reply:
I tend not to like moving literal strings to constants unless they are used multiple places in the file since it's harder to read the code. I find myself always having to scroll up somewhere to see what the value really is.